src/GenAPI.Desktop/Microsoft.DotNet.BuildTools.GenAPI.targets

```diff
@@ -2,9 +2,12 @@
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <GenAPITargetPath Condition="'$(GenAPITargetPath)' == ''">$(TargetDir)$(TargetName).cs</GenAPITargetPath>
-    <_GenAPIPath>$(MSBuildThisFileDirectory)\..\tools\GenApi.exe</_GenAPIPath>
+    <_GenAPIPath>$(MSBuildThisFileDirectory)\..\tools\GenAPI.exe</_GenAPIPath>
 
-    <!-- Sequence ourselves last in PrepareForRun 
+    <_GenAPICommand Condition=" '$(OS)' == 'Windows_NT'">"$(_GenAPIPath)"</_GenAPICommand>
+    <_GenAPICommand Condition=" '$(OS)' != 'Windows_NT'">mono --runtime=v4.0.30319 "$(_GenAPIPath)"</_GenAPICommand>
```

Comment:
@weshaggard this is in GenAPI.Desktop which only targets .NET Framework, so relying on mono as the "Desktop" runtime outside of Windows sounds fine to me.

The `--runtime=v4.0.30319` is unnecessary though, unless you have a very old Mono (and GenAPI probably wouldn't work there anyway).